WDAC-Policy-Wizard/app/MSIX/Recommended_Driver_Blocklist.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
should this file be sourced via http request to keep it in sync with the public docs version?